src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -264,7 +266,23 @@ class BinaryCompactObject : public DomainCreator<3> {
         "The time dependence of the moving mesh domain."};
   };
 
-  using options = tmpl::list<
+  template <typename BoundaryConditionsBase>
+  struct InnerBoundaryCondition {
+    static std::string name() noexcept { return "InnerBoundaryCondition"; }
+    static constexpr Options::String help =
+        "Options for the inner boundary conditions.";
```

Comment:
To make clearer this applied to all excision regions, I suggest one of,
- append a sentence like "This boundary condition will be applied to all inner excision boundaries", OR
- "inner" -> "excision region"